tensorflow/python/compiler/tensorrt/test/unary_test.py

```diff
@@ -98,5 +99,55 @@ def ExpectedEnginesToBuild(self, run_params):
     return ["TRTEngineOp_0"]
 
 
+# Due to a leak in TF ResourceManager, where we store the TRT engines cache,
+# we sometimes observe test failures. We initially put the following test
+# in trt_convert_test.py and we observed those failures. The symptoms vary
```

Comment:
"We initially put the following test in trt_convert_test.py and we observed those failures."

Now that you put the test in compiler/tensorrt/test/ not trt_convert_test.py, do you still observe the test failure? If not, do you know why putting the test  here makes such a difference?

Reply:
Thanks @bixia1 and @aaroey for the initial review! I have investigated this further and found that the problem also appears during calibration in the master branch. I have created Issue #36304 to describe the problem in more detail. 

Putting the test here made a difference because this is the only test in this file that uses native segment execution. This way we do not observe test failure anymore.